x/stake/keeper/validator_test.go

```diff
@@ -93,6 +93,30 @@ func TestUpdateValidatorByPowerIndex(t *testing.T) {
 	require.True(t, keeper.validatorByPowerIndexExists(ctx, power))
 }
 
+func TestZeroPowerUnbonded(t *testing.T) {
```

Comment:
name should maybe reference slashing - this is a test for 0-pow when slashing occurs not unbonding

Reply:
Updated to `TestSlashToZeroPowerRemoved`